packages/fxa-settings/src/components/UnitRowRecoveryKey/index.tsx

```diff
@@ -54,6 +54,12 @@ export const UnitRowRecoveryKey = () => {
           ? l10n.getString('rk-action-remove', null, 'Remove')
           : l10n.getString('rk-action-create', null, 'Create')
       }
+      disabled={!account.hasPassword}
+      disabledReason={l10n.getString(
+        'rk-no-pwd-action-disabled-reason',
+        null,
+        'Set a password to use Sync and certain account security features.'
```

Comment:
I changed this in my PR to "Firefox Sync" since that's how we refer to it it user-facing bits. I've also called the ID `security-set-password` - maybe copy that so I don't have to change it?

Alternatively, I think this could be more descriptive with something like, "Set a password before creating a recovery key," and then, "Set a password before adding two-step authentication" for the other?